construction.tex

```diff
@@ -1,8 +1,13 @@
 \section{Construction}\label{section:construction}
 
+We implement the blockchain address protocol for Bitcoin P2PKH transactions. The implementation is parameterized by a secure signature scheme $S$, and a hash function $H$.
```

Comment:
We don't implement it, we just outline it here – it is already implemented by Bitcoin itself, after all.

Reply:
We should spell out p2pkh as "pay to public key hash" the first time it is mentioned. Please ensure also that P2PKH is capitalized throughout.